src/collection.jl

```diff
@@ -142,3 +152,43 @@ function images end
         @yield FITS(row.path)[row.hdu]
     end
 end
+
+
+@doc raw"""
```

Comment:
Out of curiosity, why using `@doc` and the raw string?

Reply:
Because of this `r"fits(\.tar\.gz)?"i` in documentation.

Reply:
```julia
julia> """
       r"fits(\\.tar\\.gz)?"i
       """
"r\"fits(\\.tar\\.gz)?\"i\n"
```